packages/vm-config/decentral-itest-orchestration-config.json

```diff
@@ -53,6 +53,20 @@
                 }
             ]
         },
+        {
```

Comment:
Kind of surprised this wasn't here already. The PSM in production has USDC as an option